clock/scheduledJobs/jobCleanup.js

```diff
@@ -53,46 +69,45 @@ function execute(batchSize, delay) {
 
     // get n jobs
     return kue.Job.rangeByStateAsync('complete', from, to, 'asc')
-    .catch((err) => {
-      console.log('Error getting completed jobs from queue', err);
-      return Promise.reject(err);
-    })
+      .catch((err) => {
+        console.log('Error getting completed jobs from queue', err);
+        return Promise.reject(err);
+      })
+
+      // delete n jobs
+      .then((jobs) =>
+        Promise.all(jobs.map((job) => {
+          if (delay > 0 && now - job.updated_at < delay) {
+            return 'skipped';
+          }
 
-    // delete n jobs
-    .then((jobs) =>
-      Promise.all(jobs.map((job) => {
-        if (delay > 0 && now - job.updated_at < delay) {
-          return 'skipped';
-        } else {
           return job.removeAsync()
-          .catch((err) => {
-            console.log(`Error removing job ${job.id}`, err);
-            return Promise.reject(err);
-          });
-        }
-      }))
-    )
+            .catch((err) => {
+              errorJobCount++;
+              return Promise.reject(err);
+            });
+        }))
+      )
+
+      // count skipped jobs
+      .then((results) => {
+        const skippedCount = results.reduce((count, result) =>
+            result === 'skipped' ? count + 1 : count,
```

Comment:
`count + 1` should be `count++` ?

Reply:
I agree, didn't change because it was already here and the tests are showing the right value. Will change that anyway.

Reply:
Actually should be ++count or count+1, otherwise reduce will generate: 1.

``
var numbers = [1,1,1,1,1];

const quantityInArray = numbers.reduce((count, result) => ++count);
console.log(quantityInArray); // 4

const quantityInArray = numbers.reduce((count, result) => count++);
console.log(quantityInArray); // 1

const quantityInArray = numbers.reduce((count, result) => count+1);
console.log(quantityInArray); // 4
``